modules/ROOT/pages/runtime-fabric-logs.adoc

```diff
@@ -22,83 +22,262 @@ Runtime Fabric enables you to specify the level of severity of the message writt
+. From Ops Center, navigate to *Logging*.
+. Select *Kubernetes*.
+. Select the *Pods* tab.
+. In the drop-down list adjacent to the search input area, select the ID of the environment where the application is deployed.
+. Locate the pod name that starts with the name of your application.
+. Select that pod name and then select *Logs*.
+
+The *Logs* tab is displayed with a filter applied to your application.
 
+[NOTE]
+To view the latest logs, select *Refresh*.
+
+=== Filters
 
-== Viewing Logs
+There are two levels of filters for drilling down in the logs:
 
-Ops Center shows a stream of logs outputted by applications and services running on Runtime Fabric. Navigate to "Logging" on the left of the page to view the logging interface.
+* The `_Containers_` filter filters by container name.
+* The `_Pods_` filter filters by pod name.
++
+This is useful for specifying application names that are followed by a wildcard (`%`).
 
-=== View Logs from an Application
+== Forward Logs to External Services
 
-With Ops Center, you can view the logs from a deployed application. This can be useful in cases where log forwarding is not set up.
+Runtime Fabric supports log forwarding to:
 
-. On Ops Center, click on Kubernetes on the left sidebar.
-. Click on the Pods tab.
-. Select the environment ID where the application was deployed on the right dropdown, near the search input.
-. Find the Pod name which begins with the name of your application.
-. Click on the Pod name and select "Logs".
+* Anypoint Monitoring
+* Third-party logging solutions using Fluent Bit log forwarding output plugins:
 
-The page should redirect to the Logs tab with a filter applied to your application.
+** Azure Log Analytics
+** Elasticsearch
+** Graylog Extended Log Format (GELF)
+** Splunk
 
 [NOTE]
-To view the latest logs, click the "Refresh" button on the upper right portion of the page.
+Only one external log forwarding configuration is supported at any one time. However, you can enable Anypoint log forwarding and an external log forwarding service at the same time.
 
-=== Filters
+=== Enable Log Forwarding
 
-There are two levels of filters to help drill down on the logs to make visible:
+. Install and configure the applicable third-party logging solution. Verify that you have the required resources allocated. Refer to the documentation for your third-party logging solution for details. 
+. From Anypoint Platform, select *Runtime Manager*.
+. Select *Runtime Fabrics* in the sidebar navigation.
+. Select the applicable Runtime Fabric based on the name used during installation.
+. Select the *Log Forwarding* tab.
+. Select the *Forward to Anypoint Monitoring (Requires Titanium)* checkbox if you have a Titanium subscription and you want to forward logs to Anypoint Monitoring.
+. Select the *Forward to other logging provider* checkbox to forward logs using a Fluent Bit log forwarding output plugin.
```

Comment:
@yianL It seems our UI wording changed to `Forward application logs to an external service`?

Reply:
Yes, you're right.

Reply:
Updated this.